README.md

```diff
@@ -33,29 +66,23 @@ $session->getPage()->findLink('Chat')->click();
 
 Please refer to [MinkExtension-example](https://github.com/Behat/MinkExtension-example) for an executable example.
 
-## Installation
-
-``` bash
-composer require behat/mink mink/webdriver-classic-driver
-```
-
 ## Testing
 
-1. Start WebDriver
-    1. If you have Docker installed, run
-    ```bash
-    docker run -p 4444:4444 selenium/standalone-firefox:4.18.1
-    ```
-    2. If you do not have Docker, but you have Java
-    ```bash
-    curl -L https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.18.0/selenium-server-4.18.1.jar > selenium-server-4.18.1.jar
-    java -jar selenium-server-4.18.1.jar standalone
-    ```
+1. Start WebDriver:
+    1. If you have Docker installed, run:
+        ```shell
+        docker run -p 4444:4444 selenium/standalone-firefox:4.18.1
+        ```
+    2. If you do not have Docker, but you have Java, run:
+        ```shell
+        curl -L https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.18.0/selenium-server-4.18.1.jar > selenium-server-4.18.1.jar
+        java -jar selenium-server-4.18.1.jar standalone
+        ```
 2. Start WebServer by running
-    ``` bash
+    ```shell
     ./vendor/bin/mink-test-server
     ```
-3. Start PhpUnit
-    ```bash
+3. Run PHPUnit
+    ```shell
     ./vendor/bin/phpunit -v --coverage-clover=coverage.clover
```

Comment:
Please remove all the arguments:

* `--coverage-clover` - I doubt, that developers would require coverage to be collected on each test run;
* `-v` - not sure what it does.

Reply:
Right, how did we end up with that in the first place? 😄